pkg/iptables/iptables.go

```diff
@@ -138,3 +140,51 @@ func InsertUnique(ipt Interface, table, chain string, position int, ruleSpec []s
 
 	return nil
 }
+
+// UpdateChainRules ensures that the rules in the list are the ones in rules, without any preference for the order,
+// any stale rules will be removed from the chain, and any missing rules will be added.
+func UpdateChainRules(ipt Interface, table, chain string, rules [][]string) error {
```

Comment:
This deserves a unit test ;-)

Reply:
You're right, since now ipt Interface really allows that. I'm adding that on a follow up PR, starting now. 

Thanks for the quick review.